<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindTandemsDialog</class>
 <widget class="QDialog" name="FindTandemsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Tandems</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/repeat_finder/repeats_image</normaloff>:/repeat_finder/repeats_image</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Base</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="paramsBox">
         <property name="title">
          <string>Tandem finder parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Tandem preset</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="repeatLenComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <property name="minimumContentsLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Min period</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="minPeriodBox">
            <property name="suffix">
             <string> n</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Max period</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="maxPeriodBox">
            <property name="suffix">
             <string> n</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="value">
             <number>1000000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="regionBox">
         <property name="title">
          <string>Region to process</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="rangeSelectorLayout"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="annotationsWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Advanced parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="minRepeatsBox">
            <property name="prefix">
             <string>x</string>
            </property>
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="overlappedTandemsCheckBox">
            <property name="text">
             <string>Show overlapped tandems</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Minimum tandem size</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="algoComboBox">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <property name="minimumContentsLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="minTandemSizeBox">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
            <property name="value">
             <number>9</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Algorithm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Minimum repeat count</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>repeatLenComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>FindTandemsDialog</receiver>
   <slot>presetSelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minPeriodBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FindTandemsDialog</receiver>
   <slot>minPeriodChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxPeriodBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FindTandemsDialog</receiver>
   <slot>maxPeriodChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>algoComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>FindTandemsDialog</receiver>
   <slot>customization()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minTandemSizeBox</sender>
   <signal>editingFinished()</signal>
   <receiver>FindTandemsDialog</receiver>
   <slot>customization()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minRepeatsBox</sender>
   <signal>editingFinished()</signal>
   <receiver>FindTandemsDialog</receiver>
   <slot>customization()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxPeriodBox</sender>
   <signal>editingFinished()</signal>
   <receiver>FindTandemsDialog</receiver>
   <slot>customization()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minPeriodBox</sender>
   <signal>editingFinished()</signal>
   <receiver>FindTandemsDialog</receiver>
   <slot>customization()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FindTandemsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindTandemsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>presetSelected(int)</slot>
  <slot>minPeriodChanged(int)</slot>
  <slot>maxPeriodChanged(int)</slot>
  <slot>customization()</slot>
 </slots>
</ui>
